### DISCUSSION TITLE
Убрать звук падения гильзы при взятии оружия

### DISCUSSION THREAD

**Discussion by KillerBunny031** [octothorp-team/public-dobrograd#738] [Предложения]
В общем-то вся суть написана в названии. Я лично не понимаю зачем вообще этот звук. Персонаж настолько неуклюжий, что каждый раз роняет гильзу? Не думаю. Сейчас этот звук может использоваться только для того, чтобы слышать как кто-то берет оружие через стену и не идти туда. Ну, как бы не более. Поэтому предлагаю убрать его. Ну, потому что зачем он.

**Comment by aniastria** [comment]
Также он есть когда нет патрон в пистолете.

**Reply by SnakeLuck** [reply]
Чего? Ты хоть понял что ты написал? Этот звук проигрывается каждый раз когда ты достаешь ЛЮБОЕ огнестрельное оружие, и то сколько в нем патронов ничего не меняет

**Reply by Kyper1** [reply]
> Чего? Ты хоть понял что ты написал? Этот звук проигрывается каждый раз когда ты достаешь ЛЮБОЕ огнестрельное оружие, и то сколько в нем патронов ничего не меняет

А ты понял, что написал? Перечитай его сообщение 10 раз и подумай над своим столько же.
